cassandra/src/main/scala/geotrellis/spark/io/cassandra/CassandraLayerHeader.scala

```diff
@@ -0,0 +1,36 @@
+package geotrellis.spark.io.cassandra
+
+import geotrellis.spark.io.LayerHeader
+
+import spray.json._
+
+case class CassandraLayerHeader(
+  keyClass: String,
+  valueClass: String,
+  tileTable: String
```

Comment:
Yep, that's true, that's a good idea to store namespace as a part of header information.